src/handle.js

```diff
@@ -4,14 +4,18 @@ const net = require('net');
 
 let handler = (func, callback) => {
```

Comment:
Out of interest, is this more efficient? Better at handling encodings? I am not a Node expert so I don't know why strings suck. ;)